tests/org.eclipse.n4js.hlc.integrationtests/pom.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016 NumberFour AG.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+  NumberFour AG - Initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.n4js</groupId>
+		<artifactId>org.eclipse.n4js.tests</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<artifactId>org.eclipse.n4js.hlc.integrationtests</artifactId>
+	<name>org.eclipse.n4js.hlc.integrationtests</name>
+	<packaging>eclipse-test-plugin</packaging>
+	<description>Test the headless N4js compiler bundled into one jar.</description>
+
+	<properties>
+		<npmLocalRegistry>http://localhost:4873</npmLocalRegistry>
+	</properties>
+	<dependencies>
+	<!-- pull jar for integration tests on the jar -->
+		<dependency>
+			<groupId>org.eclipse.n4js</groupId>
+			<artifactId>org.eclipse.n4js.hlc</artifactId>
+			<classifier>jar-with-dependencies</classifier>
+			<version>0.0.1-SNAPSHOT</version>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<!-- copy the executable jar to this run -->
```

Comment:
give some more explanation of what is happening here

Reply:
done